server/src/operators/typo_operator.rs

```diff
@@ -650,13 +660,20 @@ fn correct_query_helper(tree: &BkTree, query: String, options: &TypoOptions) ->
     }
 
     if corrections.is_empty() {
-        None
+        CorrectedQuery {
+            query: Some(query),
+            corrected: false,
+        }
     } else {
-        let mut corrected_query = query.to_string();
+        let mut corrected_query = query.query.clone();
```

Comment:
Behavior is correct, but it shouldn't always happen. `TypoOptions` needs another param, which defaults to True, such that the user can turn this behavior off if they want to.